pom.xml

```diff
@@ -234,7 +246,90 @@
                     </to>
                 </configuration>
             </plugin>
+
+
+            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
+                <!-- RELEASE_VERSION -->
+                <version>5.2.1</version>
+                <!-- /RELEASE_VERSION -->
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${project.basedir}/src/main/resources/api-docs.yml</inputSpec>
+                            <modelPackage>org.springframework.samples.petclinic.dto</modelPackage>
+                            <generatorName>spring</generatorName>
+                            <library>spring-boot</library>
+
+                            <!-- Only enabling model generation -->
+                            <modelNameSuffix>Dto</modelNameSuffix>
+                            <generateApiTests>false</generateApiTests>
+                            <generateModelDocumentation>false</generateModelDocumentation>
+                            <generateModelTests>false</generateModelTests>
+                            <generateApis>false</generateApis>
```

Comment:
I propose to generate the API (interface). Spring MVC controllers just have to implement it. This if the Open Contract change, the compiler will fail.

Reply:
OK I will do that

Reply:
Postpone to https://github.com/spring-petclinic/spring-petclinic-rest/issues/56